dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -352,6 +352,34 @@
 			zephyr,memory-region = "OPTION_SETTING_S";
 			status = "okay";
 		};
+
+		canfd_global: canfd_global@40380000 {
```

Comment:
Do we not need to support other boards?
We've already added many boards.
You should try to apply this to them as well if you can.

Reply:
I have added support for EK-RA8M1, EK-RA8D1 and MCK-RA8T1. Other boards that needed additional code to support will be added later.